bin/test.py

```diff
@@ -7,19 +7,11 @@
 TEST_APP_BUCKET_PREFIX='awslabs-startup-kit-templates-test-eb-v1-tmp-'
 TEST_APP_SOURCE_BUCKET='awslabs-startup-kit-templates-test-eb-v1'
 TEST_PYTHON_APP_KEY='eb-python-flask.zip'
-STACK_BUCKET='awslabs-startup-kit-templates-deploy-v1'
-EB_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc-bastion-eb-rds.cfn.yml'
-VPC_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc.cfn.yml'
-VPC_BASTION_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc-bastion.cfn.yml'
+EB_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v2/vpc-bastion-eb-rds.cfn.yml'
+VPC_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v2/vpc.cfn.yml'
+VPC_BASTION_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v2/vpc-bastion.cfn.yml'
```

Comment:
nitpick: consider "DRY"ing up the URLs by factoring out base URL and path to bucket into a separate variable that appear to be sensitive to version changes.

```
https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v2/
```